CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 - [#4721](https://github.com/influxdb/influxdb/pull/4721): Export tsdb.InterfaceValues
 - [#4681](https://github.com/influxdb/influxdb/pull/4681): Increase default buffer size for collectd and graphite listeners
 - [#4659](https://github.com/influxdb/influxdb/pull/4659): Support IF EXISTS for DROP DATABASE
+- [#4685](https://github.com/influxdb/influxdb/pull/4685): Heal Raft Cluster
```

Comment:
FWIW, I think this could be a bit more descriptive. E.g. "Cluster auto-promotes data node to Raft nodes on loss of Raft node". Just a suggestion.